src/main/generated/assets/minelabs/lang/en_us.json

```diff
@@ -26,11 +26,11 @@
   "block.minelabs.lab.lab_sink": "",
   "block.minelabs.lab.microscope": "",
   "block.minelabs.lab.tuberack": "",
-  "block.minelabs.lab_cabin": "Lab Cabin",
-  "block.minelabs.lab_center": "Lab Center",
-  "block.minelabs.lab_corner": "Lab Corner",
-  "block.minelabs.lab_drawer": "Lab Drawers",
-  "block.minelabs.lab_sink": "Lab Sink",
+  "block.minelabs.lab_cabin": "Cabin",
```

Comment:
Isn't it lab.lab_cabin (subdirectory lab for all of them)

Reply:
Also why are we removing "Lab" from them ?
This was added to avoid confusion if players have different mods that also add cabins/drawers. (ex. Cooking for Blockheads)